src/cli/repo/s3.go

```diff
@@ -129,12 +129,11 @@ func initS3Cmd(cmd *cobra.Command, args []string) error {
 		return Only(ctx, errors.Wrap(err, "Retrieving s3 configuration"))
 	}
 
-	if strings.Contains(s3Cfg.Endpoint, "http") {
-		invalidEndpointErr := `endpoint doesn't support fullpath. 
-		pass --disable-tls flag to use http:// instead of default https://`
+	if strings.HasPrefix(s3Cfg.Endpoint, "http") {
```

Comment:
While this might be OK, I feel like being more explicit here by including `://` might be a safer option.
```suggestion
	if strings.HasPrefix(s3Cfg.Endpoint, "http://") || strings.HasPrefix(s3Cfg.Endpoint, "https://") {
```

Reply:
Then I need to check for both http and https. Will do it to stay on safer side